<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Caducados</class>
 <widget class="QDialog" name="Caducados">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>616</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="QLineEdit" name="lineEditCodigo"/>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="spinBox">
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1" colspan="4">
    <widget class="QLineEdit" name="lineEditDescripcion"/>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Fecha</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Descripci&amp;ón</string>
     </property>
     <property name="buddy">
      <cstring>lineEditDescripcion</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>C&amp;ódigo</string>
     </property>
     <property name="buddy">
      <cstring>lineEditCodigo</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="2">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Cantidad</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QComboBox" name="comboBox"/>
   </item>
   <item row="5" column="1">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Procesar</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QPushButton" name="pushButton_2">
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="autoDefault">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditCodigo</tabstop>
  <tabstop>spinBox</tabstop>
  <tabstop>pushButton</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>comboBox</tabstop>
  <tabstop>lineEditDescripcion</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_2</sender>
   <signal>clicked()</signal>
   <receiver>Caducados</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>331</x>
     <y>129</y>
    </hint>
    <hint type="destinationlabel">
     <x>528</x>
     <y>120</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
